Simulator/components/GatherDataFrame.jsx

```diff
@@ -0,0 +1,21 @@
+import React, { Component, PropTypes } from 'react';
+
+class GatherDataFrame extends Component {
+  render() {
+    return (
+      <iframe
+        style={{ display: 'none' }}
+        src={this.props.wdcUrl}
```

Comment:
I noticed that the url bar has an on-changed event handler which sets the wdcUrl prop. Does that mean every time I enter a key stroke in the url bar, the source of this iframe changes?